docs/user_guide/backup/index.md

```diff
@@ -6,224 +6,75 @@ sidebar: sidebars/documentation.html
 
 # Backup
 
-## Prepare Backup
-
-In the following sections we will be using `/backup` directory as our targed directory that will hold backup archives. Let's prepare it now.
+In the following sections we will be using `/tmp/backup` directory as our targed directory that will hold backup archives.
 
 All the following commands are executed under `root` system account.
 
-```
-export BDIR=/backup
-umask 0027
-mkdir -p $BDIR
-chgrp postgres $BDIR
-chmod 770 $BDIR
-cd $BDIR
-```
-
-## Backup system files
+## Option One: Offline repositories backup
 
-It is necessary to backup configuration files and important data files. Being in the `$BDIR` execute:
+By default, the whole Katello instance will be turned off completely for the entire backup.
 
 ```
-tar --selinux -czvf config_files.tar.gz \
-/etc/foreman-installer \
-/etc/foreman \
-/etc/elasticsearch \
-/etc/candlepin \
-/etc/pulp \
-/etc/pki/katello \
-/etc/pki/pulp \
-/etc/gofer \
-/etc/qpid \
-/etc/qpidc.conf \
-/etc/qpidd.conf \
-/etc/sysconfig/katello \
-/etc/sysconfig/foreman \
-/etc/sysconfig/elasticsearch \
-/root/ssl-build \
-/var/www/html/pub \
-/usr/share/katello/candlepin-cert.crt
-
-tar --selinux -czvf elastic_data.tar.gz /var/lib/elasticsearch
+katello-backup /tmp/backup
 ```
 
-Please note some of these directories are created after `foreman-installer` is executed for the first time.
-
-## Repositories
-
-For backing up Pulp repositories we will not use compression program, because RPM files have usually low compression ration and depending on the instance size the `pulp_data.tar` archive can be quite big.
+## Option Two: Online repositories backup
 
-### Option One: Online repositories backup
-
-There are two options to backup repositories. First one is to do checksum of all timestamps, do the backup and do the checksum again. If both checksums match, the online backup is correct and can be used. Otherwise you need to start over. Pulp does not support online repositories backup.
+Backing up the repositories can take an extensive amount of time. You can perform a backup while online. In order for this procedure to succeed, you must not change or update the repositories database until the backup procedure is complete. Thus, you must avoid publishing, adding, or deleting content views, promoting content view versions, and adding, deleting, or syncing repositories during this time. To perform an online-backup of the repositories, run:
 
 ```
-find /var/lib/pulp -printf '%T@\n' | md5sum
-tar --selinux -cvf pulp_data.tar /var/lib/pulp /var/www/pub
-find /var/lib/pulp -printf '%T@\n' | md5sum
+katello-backup /tmp/backup --online-backup
```

Comment:
my brain wants `katello-backup --online-backup /tmp/backup` here. I know ruby does not care, just a habbit :)